.travis.yml

```diff
@@ -48,14 +48,13 @@ install:
   # pyenv's shims directory is not in the PATH by default.
   - export PATH="$(pyenv root)/shims:${PATH}"
   # These are the Python versions that we want to test on.
-  - pyenv install --skip-existing ${PY34}
   - pyenv install --skip-existing ${PY35}
   - pyenv install --skip-existing ${PY36}
   - pyenv install --skip-existing ${PY37}
-  - pyenv global ${PY37} ${PY36} ${PY35} ${PY34} system
+  - pyenv global ${PY37} ${PY36} ${PY35} system
   # tox is not installed by default in some older Travis environments.
   - python3.6 -m pip install tox
   - make -f Makefile
 script:
   - export DYLD_LIBRARY_PATH=`pwd`/tests/objc
-  - python3.6 -m tox -e "py{34,35,36,37}-default,flake8"
+  - python3.6 -m tox -e "py{35,36,37}-default,flake8"
```

Comment:
Now that Python 3.7 is the newest installed version, should we change this call to use `python3.7`? (It probably doesn't make much of a difference in practice, since that's only the Python version used to run `tox` itself. The tests are run in separate virtualenvs with the proper Python versions no matter what.)

Alternatively we could remove the explicit Python call here altogether and let `tox` get called via the PATH (which *should* work because we call `pyenv global ...` earlier).

Reply:
Explicit is better than implicit.